### DISCUSSION TITLE
Show controls panel when using both stories and docs?

### DISCUSSION THREAD

**Discussion by kayleecodes1** [storybookjs/storybook#15453] [Help] [answered]
I'm trying to utilize Storybook on my current project for both development AND documentation. Ideally I would use the Canvas tab as normal for development, with adjustable args to control props. And then also use the Docs tab for markdown documentation, props documentation, etc.

Everything's working at the moment but I must be missing something that has changed about the controls/actions addons. **Is there any way to show the control panel as normal on the Canvas tab?** The only way I've been able to control the args of a story is when it is rendered inside of the Docs page (a control table shows up underneath). But I've been completely unable to get the old controls/actions panel to show up anywhere. It's getting very frustrating. Has this just been disabled entirely in the new versions?

<img width="1470" alt="Screen Shot 2021-07-01 at 10 57 30 AM" src="https://user-images.githubusercontent.com/3307853/124145888-33bf8280-da5b-11eb-908f-09c839b8f578.png">
<img width="1489" alt="Screen Shot 2021-07-01 at 10 57 37 AM" src="https://user-images.githubusercontent.com/3307853/124145890-34581900-da5b-11eb-902a-9950a9dd6951.png">

**Comment by shilman** [comment, marked as answer]
Try this: https://github.com/storybookjs/storybook/issues/8383#issuecomment-541562349

**Reply by kayleecodes1** [reply]
Thank you so much! I knew something was wrong! I'm not sure how I missed that when I was searching for answers. I am forever grateful 👍
